app/code/Magento/Bundle/Model/Product/Type.php

```diff
@@ -1272,6 +1277,40 @@ protected function checkIsAllRequiredOptions($product, $isStrictProcessMode, $op
         }
     }
 
+    /**
+     * Validate Options for Radio and Select input types
+     *
+     * @param \Magento\Bundle\Model\ResourceModel\Option\Collection $optionsCollection
```

Comment:
do "use Magento\Bundle\Model\ResourceModel\Option\Collection" and only use "Collection"